ci-operator/config/openshift-kni/lifecycle-agent/openshift-kni-lifecycle-agent-main.yaml

```diff
@@ -97,6 +97,21 @@ tests:
       SEED_IMAGE_TAG_FORMAT: presubmit
     workflow: openshift-image-based-upgrade-presubmit
   timeout: 4h0m0s
+- as: ibi-e2e-flow
+  cluster: build02
+  optional: true
+  skip_if_only_changed: ^docs/|LICENSE|OWNERS|\.md$|^hack$
+  steps:
+    cluster_profile: aws
+    dependencies:
+      LCA_PULL_REF: pipeline:lifecycle-agent-operator
+    env:
+      OCP_BASE_VERSION: "4.15"
```

Comment:
4.16 ci OCP images are not currently working for IBI so I set main to use 4.15 ocp images

Reply:
Whats the flow of adding a TODO / Task here to revisit this?

Reply:
I'm adding a bug to my epic as we speak

Reply:
https://issues.redhat.com/browse/OCPEDGE-1104